zerver/lib/events.py

```diff
@@ -832,6 +834,8 @@ def name(sub: Dict[str, Any]) -> str:
             user_status.pop(user_id, None)
 
         state['user_status'] = user_status
+    elif event['type'] == 'has_zoom_token':
+        state['has_zoom_token'] = event['value']
```

Comment:
Probably we want a `video_calls` data structure, with this a key, and use the `update_dict` event operation.